src/mango/src/mango_httpd.erl

```diff
@@ -225,14 +226,15 @@ start_find_resp(Req) ->
     chttpd:start_delayed_json_response(Req, 200, [], "{\"docs\":[").
 
 
-end_find_resp(Acc0) ->
-    #vacc{resp=Resp00, buffer=Buf, kvs=KVs, threshold=Max} = Acc0,
+end_find_resp(Req, Acc0) ->
+    #vacc{resp=Resp00, buffer=Buf, kvs=KVs0, threshold=Max} = Acc0,
     {ok, Resp0} = chttpd:close_delayed_json_object(Resp00, Buf, "\r\n]", Max),
+    {ok, KVs1} = mango_plugin:after_find(Req, Resp0, KVs0),
```

Comment:
should we move this after the `FinalAcc` below?

Reply:
Then we would not be able to change `KVs`.